plugins/chat/spec/jobs/regular/chat/auto_join_channel_batch_spec.rb

```diff
@@ -1,253 +0,0 @@
-# frozen_string_literal: true
```

Comment:
other specs are checking that their actions are enqueing this job and/or are seeing the effects of it